src/main/java/frc/robot/components/Vision.java

```diff
@@ -41,8 +40,8 @@ public void update() {
 
                     if(NewData.hoopStatus == 1) {
                         NewData.hx = -(Double.parseDouble(parameters[0]) - hoopCameraXOffset);
-                        NewData.hy = Double.parseDouble(parameters[1]) - hoopCameraYOffset;
-                        NewData.hr = Double.parseDouble(parameters[2]) - hoopCameraROffset;
+                        NewData.hy = Double.parseDouble(parameters[2]) - hoopCameraYOffset;
+                        NewData.hr = Double.parseDouble(parameters[1]) - hoopCameraROffset;
```

Comment:
We should clean these up to be the heading and distance to the hoop rather than x, y, and r to prevent confusing ourselves when debugging.